import.py

```diff
@@ -227,9 +230,9 @@ def find_build_command_line(
     if len(output) > 1:
         output_lines = "\n".join(map(formatcmd, output))
         print(
-            f"Error: found multiple compile commands for {rel_c_file}:\n{output_lines}\n"
-            "Please modify the build script such that if PERMUTER = 1, "
-            "only a single compile command is included.",
+            f"Error: found multiple compile commands for {rel_c_file}:\n{output_lines}\n{close_extra}"
```

Comment:
```suggestion
            f"Error: found multiple compile commands for {rel_c_file}:\n{output_lines}\n"
```
(close_extra isn't defined here, nor would its definition from the if block above make sense here)